source/lesson-1-html-and-css.html.md.erb

```diff
@@ -97,22 +101,22 @@ The `body` contains webpage content that is visible to the user.
 
 Let's start by defining the basic structure of your website.
 
-Open Atom and create a new folder for your work called `lesson-1-html-and-css`:
+Open Atom and create a new folder for your work called `lesson-1-html-and-css`. Select **File > Add Project Folder...** and select the correct folder:
 
 <details>
 <summary>Screenshots showing how to add a folder in Atom</summary>
 <img alt="Screenshot showing how to add a folder to Atom. Menu &gt; File &gt; Add Project Folder..." src="images/lesson-1/image4.png">
 <img alt="Screenshot showing how to create a new folder in the Add Project Folder dialog" src="images/lesson-1/image6.png">
 </details>
 
-Then inside this folder create a new file called `index.html`:
+Then inside this folder create a new file called `index.html` by context selecting the project folder, and selecting **New File**:
 
 <details>
 <summary>Screenshot showing how to add a file in Atom</summary>
-<img alt="Screenshot showing how to add a file in Atom. Right click on the folder in the sidebar, then select 'New File'." src="images/lesson-1/image2.png">
+<img alt="Screenshot showing how to add a file in Atom. Context select the folder in the sidebar, then select 'New File'." src="images/lesson-1/image2.png">
 </details>
 
-#### Task 1: doctype, html, head and body
+### Task 1: doctype, html, head and body
```

Comment:
I think the jump in heading levels here is evidence of a bit of an issue with the structure.

There are third level headings for `### doctype and HTML` and `### head and body tags` above, but then the `## Presentational HTML Elements` second level heading comes along and ruins things. So then we have this jump to `#### Task 1: doctype, html, head and body`, which really shouldn't be under the `## Presentational HTML Elements` heading at all.

I think we should move the "Open Atom and create a new folder..." bit up so it's directly under `## Webpage structure`, and then move these tasks up into the relevant third level headings within that section.

I appreciate that's a bit of a bigger change than the rest of this PR, so I'm happy to do it myself in a follow up if you prefer?

Reply:
@richardTowers if you think that makes sense, please do 👍